### PR TITLE
Add editable timezone on date label

### PR DESCRIPTION
**Pull request by mitogh**
*Ticket*: https://central.tri.be/issues/110991

### CODE REVIEW

File: src/modules/data/utils.js [hunk 1]
```diff
@@ -8,7 +13,11 @@
  */
 export const maybeDispatch = ( attributes, dispatch ) => ( action, key, defaultValue ) => {
 	if ( key in attributes ) {
-		dispatch( action( attributes[ key ] || defaultValue ) );
+		const useDefault = isUndefined( attributes[ key ] ) ||
```
Comment (nealfennimore) [inline src/modules/data/utils.js]: Can this be extended to all falsy values, or is it necessary for just these cases?

File: src/modules/elements/timezone/__tests__/element.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { mount } from 'enzyme';
```
Comment (nealfennimore) [inline src/modules/elements/timezone/__tests__/element.test.js]: Feel free to set this in jest globally: https://github.com/moderntribe/calendar-saas-app/blob/release/1.17.3/jest.setup.js#L40-L42

File: src/modules/data/utils.js (hunk 1 above)
Comment (mitogh) [reply src/modules/data/utils.js]: In this case we still want to use `false` as we might have toggle UI elements that have `true` as default but we want to use  the saved value such as `false`   or other values like `0` so we have to make it a bit custom

**Comment by mitogh** [comment]
Might require another set of 👀

File: src/modules/data/utils.js (hunk 1 above)
Comment (nealfennimore) [reply src/modules/data/utils.js]: That's what I figured but wanted to make sure :+1:

**Pull request merged**